### ISSUE TITLE
Make "Do you want to back up now?" notification persistent

### ISSUE THREAD

**Issue by Pointedstick** [spersson/Kup#79]
Plasma 5.16 now offers support for persistent notifications that don't time out on their own. The "Do you want to back up now?" notification seems like a perfect candidate; if it's not persistent, it's easy to miss, which doesn't help anyone.

If you make the notification persistent, you become responsible for revoking it when it no longer applies. In this case, it should probably be revoked when the user either starts the backup or opts not to.

**Comment by spersson** [comment]
It's been set to persistent all the way back to my first commit in 2011. From my memory plasma at some point started forgetting about such notifications, guessing because many applications were bad at remembering to close them when no longer relevant. The only new thing I can see is the urgency setting.  I added Urgency=Normal for all notifications now, I think that is the suitable level because "backup succeded", "backup failed" should probably be kept in notification history but not interrupt any fullscreen app.
Oh, and I also noticed a problem that if you unplug a backup USB drive before answering the persistent notification, it was not closed by Kup. Fixed now.
